gapic/templates/examples/feature_fragments.j2

```diff
@@ -0,0 +1,134 @@
+{#
+ # Copyright (C) 2019  Google LLC
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+ #}
```

Comment:
Somewhere near the top you'll also need some metadata on the sample, both the gen metadata (cf https://github.com/googleapis/gapic-generator/blob/master/src/test/java/com/google/api/codegen/gapic/testdata/py/python_library.baseline#L5202) and the sample metadata(cf. https://github.com/googleapis/gapic-generator/blob/master/src/test/java/com/google/api/codegen/gapic/testdata/py/python_library.baseline#L5207)

(I'm trying to convince devrel to use the manifest for the latter, but it may not fit into their workflow)

Reply:
The fragments file is intended to just be a selection of macros that will eventually be used by the actual sample template. It's a testing modularity thing, and there's still a bunch of stuff missing.